plug_pass_prototype_0_1.ino

```diff
@@ -180,61 +190,12 @@ void loop () {
     // Wait a bit before trying again
     Serial.println("\n\nSend a character to scan another tag!");
     Serial.flush();
-   // while (!Serial.available());
+    while (!Serial.available());
     while (Serial.available()) {
     Serial.read();
     }
     Serial.flush();    
   }
-  else 
-    {
-      Serial.println("ahhhhhhhhhhhhh");
-    }

```

Comment:
I added this else statement as the counter to the first if statement.  i don't know if it's necessary or not.

Reply:
Always nice to have traceable error statements